files/common/usr/bin/download-latest-image

```diff
@@ -43,11 +43,21 @@ done
 shift $((OPTIND - 1))
 
 [[ $# -gt 1 ]] && usage "too many arguments specified"
-[[ $# -eq 0 ]] && usage "too few arguments specified"
 
 $opt_f && rm -f "latest" >/dev/null 2>&1
 [[ -f "latest" ]] && die "file 'latest' already exists"
 
+VARIANT="$1"
+if [[ -z "$VARIANT" ]]; then
+	PLATFORM=$(cat "/var/lib/delphix-appliance/platform")
+	[[ -n "$PLATFORM" ]] || die "platform could not be determined"
+
+	VARIANT=$(cat "/usr/share/doc/delphix-entire-$PLATFORM/variant")
+	[[ -n "$PLATFORM" ]] || die "variant could not be determined"
```

Comment:
Actually, I've been thinking it would be very convenient to have a short helper script like `/usr/bin/get-appliance-variant` that would do this. I think it would be especially useful for folks outside of the systems platform team as remembering the path of the variant file is not as trivial.

Reply:
Sure, can you open a bug for this suggestion?

Reply:
I can, but I figured you might as well do it now as it would take 2 minutes and avoid all the bureaucracy of creating a new bug, then a separate PR, etc.

Reply:
Well, I'm not convinced that exposing the variant is something we should be doing. I can't recall when I needed that information, outside of this script, and I prefer to expose as little information to external consumers as is required. So, without more information, I can't say that I agree that making the variant easier to retrieve is a universally good thing to do.

Which is why I think you'd be the better person to open the bug and make the argument for creating such a script.

Reply:
okay, feel free to drop this then.